tap_hubspot/client.py

```diff
@@ -100,6 +105,17 @@ def get_next_page_token(
             next_page_token = None
         return next_page_token
 
+    def prepare_request(
+        self,
+        context: dict | None,
+        next_page_token: _TToken | None,
+    ) -> requests.PreparedRequest:
+        if self._is_incremental_search(context):
+            # Search endpoints use POST request
+            self.path = self.incremental_path
+            self.rest_method = "POST"
```

Comment:
To be safe and for completion, wdyt about adding a `full_table_path` attribute and doing this. I worry about modifying an instance attribute like this.

```suggestion
        if self._is_incremental_search(context):
            # Search endpoints use POST request
            self.path = self.incremental_path
            self.rest_method = "POST"
        else:
            self.path = self.full_table_path
            self.rest_method = "POST"
```

Reply:
I can implement a path property method to chose between incremental_path and a new full_table_path. Would that be preferred?

Reply:
@edgarrmondragon ping 😄